ARCHITECTURE.md

```diff
@@ -1,51 +1,38 @@
-This document should act as an entry point for new developers and the dc-team to get a
-bettor overview of the bots' inner workings as well as what is implemented and not at the moment.
-
-## Structure
-The bot currently uses four different chat types: `Store`, `Submit`, `Review`, and `Genesis`. 
-Upon receiving a message or a `webxdcStatusUpdate` the bot will use the map-like `chat_to_chat_type`
-database table to get the correct chat-type for each `ChatId`. 
-This happens in the `bot.rs` file where the bots' creation/configuration and starting take place. 
-After the chat type is determined, the bot passes the message or `webxdcStatusUpdate` to the respective message handlers which live in `request_handlers/<chat_type>.rs`. 
-These files contain all the logic needed to handle a message in one specific chat. 
-Most of the time the contained request handlers (be it for chat messages or `webxdcStatusUpdates`)
-receive a `State` instance which holds the bot-state (for example a connection pool to the DB)
-and the Deltachat `Context` object which is necessary to send messages receive chats etc. etc.
 
-## Chat Types
-These are the four different chat types of which only `Store` is part of the MVP:
-
-- The `Store` chat is the primary 1:1 with a bot-user who wants to use webxdcs from the store.
-  Upon receiving a 1:1 chat message _or_ when a `DC::Contact` is verified with a QR-code, 
-  the bot creates this kind of chat and sends the initial `store. xdc`. 
-  Currently implemented interaction is:
-    - Updating the store.xdc
-    - Receive updates to the app index
-    - Searching the store
-    - Downloading apps
+## The core `bot.rs` and the chats it manages 
 
-- The `Submit` chat is created by the bot when he receives a webxdc in the `store`-chat. 
-  This chat handles the submission of one singular webxdc app, identified by its `app_id`. 
-  When this chat is created, the original submitter is added to the chat, 
-  the submitted webxdc is forwarded and a helper xdc (submit-helper.xdc) is sent to the chat.
+The Bot maintains 1:1 chats with each of its users 
+who can get in contact via a QR code or by sending an arbitrary message. 
 
-- The `Review` chat is created by clicking the submit button in the `submit-helper.xdc` (previous step). 
-  When creating such a chat, the bot chooses at most 3 testers and one publisher and adds them to the chat 
-  together with one forwarded instance of the submitted webxdc as well as the `review-helper.xdc`. 
-  When all tests are complete, the publisher can click on `publish` to finally release the webxdc to the store.
+The bot currently uses two different chat types: `Store` and `Genesis`: 
```

Comment:
is this "Genesis" still applicable? IIRC the submission part got removed

Reply:
genesis is still there but not doing anything.